pkg/resource/any_test.go

```diff
@@ -49,8 +49,8 @@ metadata:
     version: 1
     phase: running
     finalizers:
-      - resource1
-      - resource2
+        - resource1
```

Comment:
not sure why there's an indentation change here?

Reply:
We had an old version of go-yaml here.

Reply:
```
commit ae27a744346343ea814bd6f3bdd41d8669b172d0
Author: Gustavo Niemeyer <gustavo@niemeyer.net>
Date:   Mon May 4 15:38:57 2020 +0100

    Make sequence indentation more usual
    
    Sequences used to be slightly different, and v3 made it slightly
    different in a slightly different way, and that's all slightly bad.
    All gone now. Sequences indent just like everything else.
```

The commit that changed that.